terraform/environment/ai_search.tf

```
@@ -0,0 +1,14 @@
+resource "azurerm_resource_group" "az_search_rg" {
+  name     = "${local.resource_prefix}-rg-search"
+  location = var.region
+  tags     = local.tags
+}
+
+resource "azurerm_search_service" "search" {
+  name                = "${local.resource_prefix}-az-search"
```

I've been trying to stick to the [Abbreviation recommendations for Azure resources](https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations) when referencing Azure services in resource names (I appreciate this isn't necessarily stated anywhere though!). So I think this should be:
```suggestion
  name                = "${local.resource_prefix}-srch"
```